vector/src/main/res/layout/fragment_live_location_map.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe rename the file to something more generic and reusable like `fragment_simple_container.xml`. I have check and a similar fragment layout does not exist yet.

Reply:
Done in [7bb73ff](https://github.com/vector-im/element-android/pull/6092/commits/7bb73ffb57e954e3a33520daf6dae0ba9f4c6665)